templates/pasted.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Pastebin</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/water.css@2/out/water.css"
+    />
+    <style>
+        body {
+          text-align: center;
+        }
+
+        #bin {
+          float: left;
+          margin-left: 25%;
+          margin-right: 10px;
+          text-align: center;
+          width: 40%;
+          height: 50px;
+          padding: 5px;
+          line-height: 40px;
+          max-width: 800px;
+          resize: none;
+          white-space: nowrap;
+          overflow-y: hidden;
+        }
+
+        #copy {
+          position: relative;
+          float: left;
+          width: 5vw;
+          min-width: 0.5in;
+          margin-top: -22.5px;
+          height: 50px;
+          padding: 0px;
+          padding-left: 5px;
+          line-height: 50px;
+          text-align: center;
+        }
+
+        input {
+          margin-left: auto;
+          margin-right: auto;
+          text-align: center;
+        }
+
+        .icon {
+          position: absolute;
+          display: inline-block;
+          margin: auto;
+          height: 25px;
+          top: 12.5px;
+          left: calc(2.5vw - (25px / 2));
+        }
+    </style>
+  </head>
+  <body>
+    <h1>Pastebin</h1>
+    <nav>
+        <a href="/">Home</a>
+        <a href="/">Download</a>
+    </nav>
+    <br>
+    <div id="content" >
+      <textarea readonly id="bin" >%CONTENT%</textarea>
+      <br>
+      <button id="copy" onclick="doCopy()" >
+        <img class="icon" src="/templates/icons/clipboard.svg" >
+      </button>
+    </div>
+    <br>
+  </body>
+  <script>
+
+    let doCopy = () => {
+      
+      let text = document.getElementById("bin").value
+      let textarea = document.createElement("textarea")
+      textarea.value = text
+      document.body.appendChild(textarea)
+      textarea.select()
+      document.execCommand("copy")
+      textarea.remove()
```

Comment:
```suggestion
      navigator.clipboard.writeText(textarea.value).then(function() {return;}, function(err) {
          console.error('Could not copy text: ', err);
      });
```

`document.execCommand` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand)